src/mscorlib/System.Private.CoreLib.csproj

```diff
@@ -385,6 +383,8 @@
   <ItemGroup>
     <Compile Include="$(BclSourcesRoot)\System\Reflection\Assembly.CoreCLR.cs" />
     <Compile Include="$(BclSourcesRoot)\System\Reflection\AssemblyName.cs" />
+    <Compile Include="$(BclSourcesRoot)\System\Reflection\AssemblyNameHelpers.cs" />
```

Comment:
yes it would be shared\system\reflection\...

When the auto-PR reflects this change into CoreRT, the PR build will break there because it will now have this type in two places. You would then go update that PR there to delete the old one to fix the PR so it can be merged.